### ISSUE TITLE
Attendance not recorded while on Share screen

### ISSUE THREAD

**Issue by jsgermar** [al-caughey/Google-Meet-Attendance#100]
It seems like the google meet attendance does not record the participants when the share screen is on. I remember that everyone should be visible to record their attendance. But when someone is screen sharing, the attendance does not update. I haven't figured out a work around on this. I hope you can help me. Thank you very much!

**Comment by AlCaughey** [comment]
Using the Grid View Fix extension(rather than Google's built-in Tiled
layout) should fix the issue.

Google's layouts only display the 'most active' participants and remove all
references to the students who do not show on your screen. Even when you
set the number of tiles, that number will be (arbitrarily) ignored if
someone shares their screen.

This problem doesn't occur when you are using GVF

**Comment by Eagle-Man** [comment]
Yeah, this is a problem with Tiled View in Meet.  If YOU are presenting, then you can still see up to 49 participants, and attendance should work fine.  But if someone ELSE is presenting, then, in order to show the presentation in a decent size, it caps the number of participants shown.  For my resolution/zoom settings, it caps at 24.  It's not an issue for me because I don't have others presenting in class much, but it certainly is annoying.  Grid View (fix) doesn't do this.
